Dockerfile

```diff
@@ -18,7 +18,6 @@ WORKDIR /doubtfire-web
 
 # Install global packages
 RUN npm --global config set user "${USER}"
-RUN npm install -g angular-cli husky
```

Comment:
@macite These aren't needed because they're listed in the package file so should be installed, I don't see them needing to be installed globally... maybe Husky if people need to commit from within image? 

But definitely not the CLI

Just running this by you unless I've missed something

Reply:
I don’t think anyone will commit from the images. Global angular may be useful from the cli in the container, though I can’t recall needing it.